voter/src/main/java/org/terracotta/voter/CustomJCommander.java

```diff
@@ -0,0 +1,110 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Terracotta Core.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.voter;
+
+import com.beust.jcommander.DefaultUsageFormatter;
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.ParameterDescription;
+import com.beust.jcommander.WrappedParameter;
+import com.beust.jcommander.internal.Lists;
+import org.terracotta.voter.options.OptionsParsing;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+import static java.lang.System.lineSeparator;
+
+public class CustomJCommander extends JCommander {
+
+  public CustomJCommander(OptionsParsing object) {
+    super(object);
+    setUsageFormatter(new UsageFormatter(this));
+  }
+
+
+  @Override
+  public Map<String, JCommander> getCommands() {
+    // force an ordering of commands by name
+    return new TreeMap<>(super.getCommands());
+  }
+
+
+  private static class UsageFormatter extends DefaultUsageFormatter {
+    private final JCommander commander;
+
+    private UsageFormatter(JCommander commander) {
+      super(commander);
+      this.commander = commander;
+    }
+
+    @Override
+    public void usage(StringBuilder out, String indent) {
+      appendOptions(commander, out, indent);
+    }
+
+
+    private void appendOptions(JCommander jCommander, StringBuilder out, String indent) {
+      out.append("Usage: ");
+      String usage = "(-server <hostname:port>,<hostname:port>... -server <hostname:port>,<hostname:port> ... | -override <hostname:port>) [-security-dir <security-dir>]";
```

Comment:
we should perhaps have the same kind of parameters as other tools. On other tools we use `-connect-to`. Is the `-server` options have the same meaning as `-connect-to` ?

Reply:
I will change it to connect-to